### ISSUE TITLE
Run on open

### ISSUE THREAD

**Issue by excalios** [yaegassy/coc-blade-formatter#10]
Is there anyway to make the formatter to run on opening a blade file or even format the blade file on save?

**Comment by yaegassy** [comment]
There is a way to format the file when saving.

**coc-settings.json**:

```jsonc
{
  // ...snip
  "coc.preferences.formatOnSaveFiletypes": ["blade"],
  // ...snip
}
```

However, `blade-formatter` is unfortunately a **"slow"** process, so I do not recommend running it when saving. :(

**Comment by excalios** [comment]
Yeah I actually already have that settings but it still doesn't format it. Honestly doesn't sure why but calling :Format :call CocAction('format') or :CocCommand bladeFormatted.run does works. Does having prettier affects it? like the blade formatter has lower priority? But reading the prettier docs it says it has a low priority though

**Comment by yaegassy** [comment]
Hmm, sorry. I don't know the cause.

**Comment by Muntader** [comment]
Same problem, did you found any solution ?

**Comment by yaegassy** [comment]
I am not able to reproduce the problem in my environment.

https://user-images.githubusercontent.com/188642/132949544-fefb8cb1-32cb-45b8-ad05-cc1887dc132f.mp4

Can you provide `minimal vimrc` and `coc-settings.json` that can reproduce the problem?

**Comment by Muntader** [comment]
@yaegassy 
I used CocInstall not Plug
My CocConfig
<pre>

{
   "languageserver": {.........},
   "coc.preferences.formatOnSaveFiletypes" : ["*"]
 }
</pre>

.vimrc for coc 
<pre>
function! s:check_back_space() abort
  let col = col('.') - 1
  return !col || getline('.')[col - 1]  =~ '\s'
endfunction

inoremap <silent><expr> <Tab>
      \ pumvisible() ? "\<C-n>" :
      \ <SID>check_back_space() ? "\<Tab>" :
      \ coc#refresh()

"" Blade type 
autocmd BufNewFile,BufRead *.blade.php set filetype=blade

</pre>

Auto format work with all files only with blade.php it's not work
Note: when  i run CocAction or Command  it's work

**Comment by yaegassy** [comment]
I can't reproduce the problem. It works with minimal configuration in my environment.

https://user-images.githubusercontent.com/188642/132953830-56356029-8887-4443-91a5-7c015bb1a672.mp4
